css-three-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-color: #1a96d9;
+}
+
+.container {
+  max-width: 850px;
+  margin: 0 auto;
+}
+
+header {
+  width: 100%;
+  font-family: Lato, sans-serif;
+  font-size: larger;
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 25px;
```

Comment:
If you're using this to add space between the `h2` and the content below, you should be using `margin-bottom` instead. `margin` is used to add space between elements